packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart

```diff
@@ -73,6 +78,7 @@ abstract class ImagePickerPlatform extends PlatformInterface {
     double? maxWidth,
     double? maxHeight,
     int? imageQuality,
+    bool forceFullMetadata = true,
```

Comment:
This is a breaking change. This needs to be reverted, as this version is broken (and we need to figure out how to make sure this kind of thing doesn't pass CI in the future).